src/elements/Input.tsx

```diff
@@ -1,4 +1,4 @@
-import * as t from '../../style/input.style';
+import * as t from '../style/input.style';
 
 export interface InputType {
```

Comment:
```suggestion
export interface InputType {
```
컨벤션에 맞지 않는 타입이름이 보이네요! 수정해주시면 좋을 것 같습니다.😊